patterns/blueprint-vpc-lattice/README.md

```diff
@@ -0,0 +1,101 @@
+3. Initialize Kubernetes secrets for cluster2
+
+```shell
+# assuming you are already in the /cluster2 folder
+chmod +x secrets.sh && ./secrets.sh
+```
+4. Deploy the kubernetes artefacts for cluster2 
+
+Deploy the datastore service to the EKS cluster in cluster2. This service fronts an Aurora PostgreSQL database and exposes REST API endpoints with path-prefixes /popular and /summary. To demonstrate canary shifting of traffic, deploy two versions of the datastore service to the cluster as shown below.
+
+```shell
+# Apply Kubernetes set of manifests to both clusters that defines the GatewayClass and Gateway resources. The Gateway API controller then creates a Lattice service network with the same name, eks-lattice-network, as that of the Gateway resource if one doesn’t exist and attaches the VPCs to the service network.
+export CLUSTER_2=cluster2
+export AWS_DEFAULT_REGION=$(aws configure get region)
+export AWS_ACCOUNT_NUMBER=$(aws sts get-caller-identity --query "Account" --output text)
+
+export CTX_CLUSTER_2=arn:aws:eks:$AWS_DEFAULT_REGION:${AWS_ACCOUNT_NUMBER}:cluster/$CLUSTER_2
+
+
+kubectl apply --context="${CTX_CLUSTER_2}" -f ./$CLUSTER_2/gateway-lattice.yml          # GatewayClass and Gateway
+kubectl apply --context="${CTX_CLUSTER_2}" -f ./$CLUSTER_2/route-datastore-canary.yml   # HTTPRoute and ClusterIP Services
+kubectl apply --context="${CTX_CLUSTER_2}" -f ./$CLUSTER_2/datastore.yml                # Deployment
+```
+
+5. Deploy the gateway lattice and the frontend service on cluster1
+
+The frontend service is configured to communicate with the datastore service in cluster1 using its custom domain name. 
+
+```shell
+export CLUSTER_1=cluster1
+export AWS_DEFAULT_REGION=$(aws configure get region)
+export AWS_ACCOUNT_NUMBER=$(aws sts get-caller-identity --query "Account" --output text)
+
+aws eks update-kubeconfig --name $CLUSTER_1 --region $AWS_DEFAULT_REGION
+
+export CTX_CLUSTER_1=arn:aws:eks:$AWS_DEFAULT_REGION:${AWS_ACCOUNT_NUMBER}:cluster/$CLUSTER_1
+
+
+kubectl apply --context="${CTX_CLUSTER_1}" -f ./$CLUSTER_1/gateway-lattice.yml   # GatewayClass and Gateway
+kubectl apply --context="${CTX_CLUSTER_1}" -f ./$CLUSTER_1/frontend.yml  # Frontend service
+```
+
+## Testing if cluster1 service could talk to cluster2 service via VPC lattice 
+
+Shell commands below uses kubectl port-forward to forward outgoing traffic from a local port to the server port 3000 on one of the pods of the frontend service, which allows us to test this use case end-to-end without needing any load balancer.
+
+```shell
+POD=$(kubectl -context="${CTX_CLUSTER_1}" get pod -n apps -l app=frontend -o jsonpath="{.items[0].metadata.name}")
+kubectl -context="${CTX_CLUSTER_1}" -n apps port-forward ${POD} 80:3000 # Port Forwarding
+
+curl -X GET http://localhost/popular/category|jq
+curl -X GET http://localhost/summary|jq # you could retry the summary to see if you get a different results from different versions
```

Comment:
1. This now introduces a new dependency required for users to run the pattern - `jq`. Do we need to use `jq` or can we just use the raw output?
2. What should users expect to see here? How do they know if the pattern is setup and working as intended?